src/sparseml/modifiers/obcq/utils/data.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+from typing import List, Tuple
+
+import torch
+from datasets import load_dataset
+from torch.nn import Module
+from transformers import AutoTokenizer, GPT2Tokenizer
+
+
+__all__ = ["get_wikitext2", "get_ptb", "get_c4", "get_openplatypus"]
```

Comment:
would be good to generalize these a bit when we can 1) to work with registry pattern so it is easy to add and reference datasets 2) to share on deepsparse side for eval